pinot-spi/src/main/java/org/apache/pinot/spi/config/table/ingestion/EnrichmentConfig.java

```diff
@@ -32,11 +32,16 @@ public class EnrichmentConfig extends BaseJsonConfig {
   @JsonPropertyDescription("Enricher properties")
   private final JsonNode _properties;
 
+  @JsonPropertyDescription("Record enricher type that is applied before complex type transformation")
+  private final boolean _preComplexTypeRecordEnricher;
```

Comment:
(minor) Suggest renaming, same for other places
```suggestion
  private final boolean _preComplexTypeTransform;
```